llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -18035,6 +18053,7 @@ RISCVTargetLowering::EmitInstrWithCustomInserter(MachineInstr &MI,
            "ReadCounterWide is only to be used on riscv32");
     return emitReadCounterWidePseudo(MI, BB);
   case RISCV::Select_GPR_Using_CC_GPR:
+  case RISCV::Select_IMM_Using_CC_GPR:
   case RISCV::Select_FPR16_Using_CC_GPR:
```

Comment:
Don't we need versions of all of these opcodes with an immediate condition? When you created RISCVISD::SELECTIMM_CC, you didn't check that the result type was integer so FP is possible.

Reply:
Deleted the new node. Doesn't seem to have this problem anymore?